src/pages/Poster.tsx

```diff
@@ -37,6 +36,12 @@ const RANDOM_MOVIE = gql`
       title
       releaseYear
       posterPath
+      randomMovies(num: 3, differentReleaseYear: true) {
```

Comment:
We fetch the random movies together with the "parent" random movie. The question screen can then simply read the latest state of the cache and get its data (with the exact same query, without any imdbId param).

Reply:
Nice, that's a good solution for our problem. And we also save a network request.